product_docs/docs/pem/9/certificates/index.mdx

```diff
@@ -0,0 +1,312 @@
+---
+title: "Managing SSL certificates"
+navTitle: "Managing SSL certificates"
+legacyRedirectsGenerated:
+  # This list is generated by a script. If you need add entries, use the `legacyRedirects` key.
+  - "/edb-docs/d/edb-postgres-enterprise-manager/user-guides/administrators-guide/8.0/managing_certificates.html"
+# This file is moved from pem_admin/04_managing_certificates
+redirects:
+- /pem/latest/managing_certificates/
+- /pem/latest/pem_admin/04_managing_certificates/
+navigation:
+- replacing_ssl_certificates
+- regenerating_agent_certificates
+---
+
+PEM uses self-signed SSL certificates:
+
+- To secure requests to the web server (Apache HTTPD) which provides the user interface and REST API
+- To secure and authenticate the PEM agent connections to the PEM backend database
+
+## Webserver certificates
+
+PEM uses self-signed SSL certificates for the Apache HTTPD server. The self-signed SSL certificates and keys for the Apache HTTPD server are generated during PEM installation.
```

Comment:
Is there one HTTPD certificate or more than one? Depending on the answer either make certificates singular in this paragraph or make it plural in the next.
```suggestion
PEM uses self-signed SSL certificates for the Apache HTTPD server. The self-signed SSL certificates and keys for the Apache HTTPD server are generated during PEM installation.
```

Reply:
I believe for Apache httpd (notice the product name is actually lower case, which I neglected to correct previously) there is only one key and one certificate so we could be more specific and use the singular form here.

When referring to agents, there is one certificate per agent (except in the documented case of sharing a single cert) so plural is appropriate.